tests/UI/data/xml/country.ts

```diff
@@ -0,0 +1,27 @@
+import {faker} from '@faker-js/faker';
+
+/**
+ * Get xml of country to put on Post/Put request
+ * @param idCountry {string|null}
+ */
+export default function getCountryXml(idCountry : string | null = null): string {
```

Comment:
Why not using CountryData rather than a simple idCountry or faker ?

That allows to rationalize all our types.

Reply:
Discussed with @Progi1984 : Xml types are different from faker or import types, can be done on another refacto